### PR TITLE
Feature/147

### PR DESCRIPTION
**Pull request by Aaron383**
Tried to add gradient background with a stretched png, looks good on my emulator let me know what you think

### CODE REVIEW

File: mobile/src/components/home/DiscoverBar.jsx
Comment (yismailuofa) [inline mobile/src/components/home/DiscoverBar.jsx:11]: Not sure if it's possible but could we move these svg definitions to their own files? Would make the code cleaner.

Comment (Aaron383) [reply mobile/src/components/home/DiscoverBar.jsx]: Should be fixed now

Comment (yismailuofa) [reply mobile/src/components/home/DiscoverBar.jsx]: LGTM, probably the best we can do unless we get approval to do solid color background.

**Review by yismailuofa — APPROVED** [review]

**Comment by dimassudjito** [comment]
@MoyinF please review and merge if you're happy with the code

**Review by MoyinF — COMMENTED** [review]
Looks good.

**Pull request merged**
